packages/core/forms/src/components/fields/__tests__/FieldTester.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div class="field-tester-container">
+    <h3>Field Tester</h3>
+
+    <KButton
+      class="tester-update-button"
+      data-testid="tester-update-button"
+      @click="handleUpdate"
+    >
+      Programmatically Update Model
+    </KButton>
+
+    <VueFormGenerator
+      :model="updatedFormModel"
+      :schema="schema"
+      @model-updated="(model: Record<string, any>, key: string) => handleModelUpdated(model, key)"
+    />
+
+    <hr>
+
+    <h3>Value Displays</h3>
+
+    <div>
+      <KLabel>
+        Form Model (setup ref)
+      </KLabel>
+      <div class="field-tester-form-model">
+        <div class="field-tester-form-model-row">
+          <KLabel class="first-col">
+            Field Key
+          </KLabel>
+          <KLabel class="second-col">
+            Field Value
+          </KLabel>
+        </div>
+        <div
+          v-for="fieldKey in Object.keys(updatedFormModel)"
+          :key="`${fieldKey}-field`"
+          class="field-tester-form-model-row"
+        >
+          <code
+            class="first-col"
+            :data-testid="`field-tester-form-model-${fieldKey}-label`"
+          >{{ fieldKey }}</code>
+          <div :data-testid="`field-tester-form-model-${fieldKey}-value`">
+            {{ updatedFormModel[fieldKey] }}
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <p>
+      <KCollapse trigger-label="Show/hide schema (prop)">
+        <code data-testid="field-tester-prop-schema"><pre>{{ JSON.stringify(schema, null, 2) }}</pre></code>
+      </KCollapse>
+    </p>
+
+    <p>
+      <KCollapse trigger-label="Show/hide model (prop)">
+        <code data-testid="field-tester-prop-model"><pre>{{ JSON.stringify(model, null, 2) }}</pre></code>
+      </KCollapse>
+    </p>
+
+    <p>
+      <KCollapse trigger-label="Show/hide modifiedModel (prop)">
+        <code data-testid="field-tester-prop-modified-model"><pre>{{ JSON.stringify(modifiedModel, null, 2) }}</pre></code>
+      </KCollapse>
+    </p>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, type PropType } from 'vue'
+import type { FormSchema } from '../../../types'
+import VueFormGenerator from '../../FormGenerator.vue'
+
+const props = defineProps({
+  schema: {
+    type: Object as PropType<FormSchema>,
+    default: () => ({
+      fields: [],
+    }),
+  },
+  model: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => ({}),
+  },
+  modifiedModel: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => ({}),
+  },
+})
+
+// verify value displays on load
+const updatedFormModel = ref<Record<string, any>>(props.model)
+
+// verify model updated reflects input events
+const handleModelUpdated = (model: Record<string, any>, key: string): void => {
+  updatedFormModel.value[key] = model
+}
+
+// verify programmatic updates to model
+const handleUpdate = (): void => {
+  updatedFormModel.value = props.modifiedModel
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Is there a reason for not using design tokens here ?

Reply:
Mostly because I was lazy and it's an internal-only tool. I'll go back and clean it up 👍🏻